src/Bibliography.js

```diff
@@ -136,13 +136,15 @@ export default class Bibliography {
 			reference.citations.push(uuid);
 		}
 
-		let parts = parseFormattedCitationSequence(text, citations);
-
-		if (parts === null) {
+		let parts;
+		if (citations.length > 1) {
 			// Sequence could not be parsed into parts directly
 			// Because not all citations are present in the output
```

Comment:
Ok this needs actual rewording not just a one word fix

Reply:
I tried to fix it. Could you please take another look?